packages/web/components/alert/toast.tsx

```diff
@@ -1,10 +1,13 @@
 import Image from "next/image";
 import { FunctionComponent } from "react";
-import { useTranslation } from "react-multi-lang";
+import { t } from "react-multi-lang";
 import { toast, ToastOptions } from "react-toastify";
 
 import { Alert, ToastType } from "./types";
 
+const tAlertField = (field: Alert["caption"] | Alert["message"]) =>
+  typeof field === "string" ? t(field) : t(...(field as Parameters<typeof t>));
```

Comment:
Is there a particular reason why we don't make `t` a parameter instead? I believe this will function as intended in a client-side rendering. However, it wouldn't work if we decide to use ssr with server components streaming